lib/features/manage_account/presentation/profiles/identities/identities_controller.dart

```diff
@@ -287,8 +304,51 @@ class IdentitiesController extends BaseController {
         AppLocalizations.of(currentContext!).you_are_changed_your_identity_successfully);
     }
 
+    _cleanUpPublicAssets(
+      success.identityId,
+      IdentityActionType.edit,
+      success.publicAssetsInIdentityArguments);
+
     _refreshAllIdentities();
   }
 
   bool get isSignatureShow => identitySelected.value != null;
+
+  void _injectCleanUpPublicAssetsInteractorBindings(Session? session, AccountId? accountId) {
+    try {
+      requireCapability(session!, accountId!, [CapabilityIdentifier.jmapPublicAsset]);
+      CleanUpPublicAssetsInteractorBindings().dependencies();
+    } catch(e) {
+      logError('$runtimeType::injectCleanUpPublicAssetsInteractorBindings(): exception: $e');
+    }
+  }
+
+  void _cleanUpPublicAssets(
+    IdentityId? identityId,
+    IdentityActionType? identityActionType,
+    PublicAssetsInIdentityArguments? publicAssetsInIdentityArguments) {
+    final session = accountDashBoardController.sessionCurrent;
+    final accountId = accountDashBoardController.accountId.value;
+
+    if (session == null
+      || accountId == null
+      || identityId == null
+      || identityActionType == null
+      || publicAssetsInIdentityArguments == null) return;
+    
+    final cleanUpPublicAssetsInteractor = getBinding<CleanUpPublicAssetsInteractor>();
```

Comment:
we should get by tag for code cleaner. @tddang-linagora wdyt?

Reply:
Done